applications/SwimmingDEMApplication/custom_python/add_custom_processes_to_python.cpp

```diff
@@ -39,7 +39,7 @@ void  AddCustomProcessesToPython(pybind11::module& m)
 
     py::class_<TransientSpatialDependantPorositySolutionBodyForceProcess, TransientSpatialDependantPorositySolutionBodyForceProcess::Pointer, Process>
     (m, "TransientSpatialDependantPorositySolutionBodyForceProcess")
-    .def(py::init< ModelPart&, const double, const double, const double, const double, const double, const double, const double, const double>())
```

Comment:
Is it necessary to keep the version with so many arguments? The one below looks more practical, with a Parameters variable, so you have more flexibility in the future, keeping the same API.

Reply:
Maybe there is another alternative, but those double are member variables that need to be defined to compute the manufactured solution.

Reply:
I was just saying that this constructor looks cooler and the other one, with many arguments, could be considered for deletion.
https://github.com/KratosMultiphysics/Kratos/blob/8736dcd98ed57802a8566f3d9df95c9ea8e3b57a/applications/SwimmingDEMApplication/custom_processes/transient_spatial_dependant_porosity_solution_body_force_process.cpp#L87

Reply:
Ok, thanks for the advise. For the moment I will keep in that way until to find out another alternative.

Reply:
> I was just saying that this constructor looks cooler and the other one, with many arguments, could be considered for deletion.
> https://github.com/KratosMultiphysics/Kratos/blob/8736dcd98ed57802a8566f3d9df95c9ea8e3b57a/applications/SwimmingDEMApplication/custom_processes/transient_spatial_dependant_porosity_solution_body_force_process.cpp#L65

The other alternative is already coded here, same file, same object. Just informing.

Reply:
Ok, I have already modified